doc/man3/RAND_add.pod

```diff
@@ -2,63 +2,75 @@
 
 =head1 NAME
 
-RAND_add, RAND_seed, RAND_status, RAND_event, RAND_screen - add
-randomness to the PRNG
+RAND_add, RAND_poll, RAND_seed, RAND_status, RAND_event, RAND_screen
+- add randomness to the PRNG
 
 =head1 SYNOPSIS
 
  #include <openssl/rand.h>
 
- void RAND_seed(const void *buf, int num);
+ int RAND_status(void);
+ int RAND_poll()
 
  void RAND_add(const void *buf, int num, double randomness);
-
- int  RAND_status(void);
+ void RAND_seed(const void *buf, int num);
 
  #if OPENSSL_API_COMPAT < 0x10100000L
- int  RAND_event(UINT iMsg, WPARAM wParam, LPARAM lParam);
+ int RAND_event(UINT iMsg, WPARAM wParam, LPARAM lParam);
  void RAND_screen(void);
  #endif
 
 =head1 DESCRIPTION
 
-RAND_add() mixes the B<num> bytes at B<buf> into the PRNG state. Thus,
-if the data at B<buf> are unpredictable to an adversary, this
-increases the uncertainty about the state and makes the PRNG output
-less predictable. Suitable input comes from user interaction (random
-key presses, mouse movements) and certain hardware events. The
-B<randomness> argument is an estimate of how much randomness is contained in
+Random nummbers are a vital part of key generation, and software-based
+generators must be "seeded" with external randomness before they can 
+used to generate a cryptographically-strong serious of psuedo-random
+numbers (CSPRNG).
+The combination of modern operating systems, which can use items such
+as interrupt jitter and network packet timings, and common hardware with
+special instructions, can generate excellent sources of seeds.
```

Comment:
I would really like to avoid calling some of those an excellent source.

I think the combination is also not of OSs but of sources.